system/views/view.py

```diff
@@ -0,0 +1,62 @@
+import tkinter as tk
+from tkinter import ttk
+import system.views.registration_page
+from system.views.login_page import LoginPage
+from system.views.user_page import UserPage
+from system.views.admin_page import AdminPage
+
+
+class View(tk.Tk):
```

Comment:
Two things could change/happen here:
1. The implementation of this class is moved to the __ main __ file outside of the system. (you'll need to rebase your branch against the master to see this file)
2. The __ main __ file will call this view and it will stay here.

Now that I'm thinking about it, it would probably be better for the __main__ to call this view. Let me know if you don't understand.